src/components/Button.js

```diff
@@ -29,5 +39,9 @@ Button.propTypes = {
     children: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
 };
 
+// Example for disabled button:
+//
+// <Button type="submit" disabled={true} variant="primary">Disabled</Button>
+//
```

Comment:
Not sure if this comment is meant to be here?

Reply:
yep, it was on purpose!